tests/kuery/test_evaluator.py

```diff
@@ -99,7 +99,7 @@ def test_cidr_match(self):
         self.assertFalse(self.evaluate('ip:10.0.0.0/8'))
 
     def test_quoted_wildcard(self):
-        self.assertFalse(self.evaluate('string:"*"'))
+        self.assertTrue(self.evaluate('string:"*"'))
```

Comment:
This should remain false because the `string` field from the document has no literal `*` in the value

`"string": "hello world",`

```suggestion
        self.assertFalse(self.evaluate('string:"*"'))
```